scarb/src/bin/scarb/args.rs

```diff
@@ -279,9 +285,9 @@ pub struct InitArgs {
     #[arg(long)]
     pub no_vcs: bool,
 
-    /// Use a Starknet Foundry Forge template.
-    #[arg(long)]
-    pub snforge: bool,
+    /// Test runner to use. Starts interactive session if not specified.
+    #[arg(long, env = "SCARB_TEST_RUNNER")]
```

Comment:
```suggestion
    #[arg(long, env = "SCARB_INIT_TEST_RUNNER")]
```

isn't this more correct? without the `INIT` part it looks like a switch that forces test runner in runtime, not project creation